src/Freya.Core/Operators.fs

```diff
@@ -17,46 +17,79 @@
 // limitations under the License.
 //----------------------------------------------------------------------------
 
-module Freya.Core.Operators
+namespace Freya.Core
 
-(* Operators *)
+/// Core combinator definitions for <see cref="Freya{T}" /> computations.
+[<CompilationRepresentation(CompilationRepresentationFlags.ModuleSuffix)>]
+module Freya =
```

Comment:
I've never used `RequireQualifiedAccessAttribute`, so if you think so, then let's do it! I should read up on it.